modules/rosa-cluster-autoscaler-cli-after.adoc

```diff
@@ -6,6 +6,11 @@
 [id="rosa-enable-cluster-autoscale-cli-after_{context}"]
 = Enable autoscaling after cluster creation with the ROSA CLI
 
+[IMPORTANT]
+====
+The following procedure is only supported for ROSA Classic clusters. To enable autoscaling after cluster creation for ROSA with HCP clusters, see link:https://docs.openshift.com/rosa/rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.html[Enabling autoscaling nodes on a cluster].
```

Comment:
Looks good! Only thing I would suggest is to change the link to _https://docs.openshift.com/rosa/rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.html#nodes-enabling-autoscaling-nodes_ or change the link text to "About autoscaling nodes on a cluster"...so the target matches the link....